Dockerfile

```diff
@@ -80,10 +89,14 @@ RUN pip2 install --no-cache-dir mock nose requests testpath && \
     pip2 uninstall -y funcsigs mock nose pbr requests six testpath && \
     pip3 uninstall -y nose requests testpath
 
+# Add a notebook profile.
+RUN jupyter notebook --generate-config && \
```

Comment:
Ok, so now I simply overwrite the file with echo. I still use `jupyter notebook --generate-config` to create the initial directory and file. Is that ok or would you like me to `mkdir` and the like instead?